### PR TITLE
Create new service update

### PR DESCRIPTION
**Pull request by MylesJarvis**
### Trello card

https://trello.com/c/q07B9Cin

### Context

- At some point we're going to remove service updates due to limited usage, but as a stopgap, and because we're shipping some major changes to how we manage users, we're putting a service update out
- This will replace the previous service update that renders on the Manage school experience dashboard, which is from March 2022. Showing that is a poor experience for users.

### Changes proposed in this pull request

- Add service update

**Dashboard**
image

image

**Update page**

image

### Guidance to review

- Does this render once you navigate to a school's dashboard ✅ 
- Does it work if you click the link from the bottom of the school dashboard ✅ 
- Does the content look ok?

### CODE REVIEW

File: data/service_updates/20240402.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Manage users within Manage school experience
```
Comment (MaxineCl) [inline data/service_updates/20240402.yml:2]: in rather than within?

File: data/service_updates/20240402.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+summary: |-
+  We've created a new feature that allows you to invite new school experience users directly from within Manage school experience.
+html_content: |-
+  <p>We’ve updated the Manage school experience service
+  so you can now add new users directly on to your school’s account.</p>
+  <h3>New users</h3>
+  <p>When you add a new user to your school profile, they can:</p>
```
Comment (MaxineCl) [inline data/service_updates/20240402.yml:9]: don't need comma here

File: data/service_updates/20240402.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    <li>advertise experience at your school</li>
+    <li>manage bookings and cancellations</li>
+    <li>add other people from your school</li>
+  </ul>
+  <h3>Adding a new user</h3>
+  <p>To add a new user:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>go to the school dashboard in your account</li>
+    <li>go to the Manage users section</li>
+    <li>go to the Manage users section</li>
```
Comment (MaxineCl) [inline data/service_updates/20240402.yml:20]: the 2nd and 3rd bullet are the same?

File: data/service_updates/20240402.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  <p>When you add a new user to your school profile, they can:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>advertise experience at your school</li>
+    <li>manage bookings and cancellations</li>
+    <li>add other people from your school</li>
+  </ul>
+  <h3>Adding a new user</h3>
+  <p>To add a new user:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>go to the school dashboard in your account</li>
```
Comment (MaxineCl) [inline data/service_updates/20240402.yml:18]: for steps we'd usually do 1, 2, 3 rather than bullets

File: data/service_updates/20240402.yml [hunk 5]
```diff
@@ -0,0 +1,21 @@
+---
+title: Manage users within Manage school experience
+summary: |-
+  We've created a new feature that allows you to invite new school experience users directly from within Manage school experience.
+html_content: |-
+  <p>We’ve updated the Manage school experience service
+  so you can now add new users directly on to your school’s account.</p>
```
Comment (MaxineCl) [inline data/service_updates/20240402.yml:7]: I'd either say now add users or you can add new users. Not sure we need both now and new (if that makes sense).

File: data/service_updates/20240402.yml [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  <ul class="govuk-list govuk-list--bullet">
+    <li>advertise experience at your school</li>
+    <li>manage bookings and cancellations</li>
+    <li>add other people from your school</li>
+  </ul>
+  <h3>Adding a new user</h3>
+  <p>To add a new user:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>go to the school dashboard in your account</li>
+    <li>go to the Manage users section</li>
```
Comment (MaxineCl) [inline data/service_updates/20240402.yml:19]: Is Manage users capped up in the service? If it is ignore this and if not I'd have it as lower case

File: data/service_updates/20240402.yml (hunk 3 above)
Comment (MylesJarvis) [reply data/service_updates/20240402.yml:20]: Dodgy copy and paste from me in VS code - good catch, thank you.

File: data/service_updates/20240402.yml (hunk 4 above)
Comment (MylesJarvis) [reply data/service_updates/20240402.yml:18]: Cool, will change to an <ol>

File: data/service_updates/20240402.yml (hunk 6 above)
Comment (MylesJarvis) [reply data/service_updates/20240402.yml:19]: yep - so leaving as is!

File: data/service_updates/20240402.yml [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+  <ul class="govuk-list govuk-list--bullet">
+    <li>advertise experience at your school</li>
+    <li>manage bookings and cancellations</li>
+    <li>add other people from your school</li>
+  </ul>
+  <h3>Adding a new user</h3>
+  <p>To add a new user:</p>
+  <ol class="govuk-list govuk-list--number">
+    <li>go to the school dashboard in your account</li>
+    <li>go to the Manage users section</li>
```
Comment (gemmadallmandfe) [inline data/service_updates/20240402.yml:20]: There isn't a section called Manage users? The link is `Add new user`

image

File: data/service_updates/20240402.yml [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+    <li>add the new user</li>
```
Comment (gemmadallmandfe) [inline data/service_updates/20240402.yml:21]: If you are changing the previous bullet to be `go to the Add new users section`, then you probably need to tweak this 

eg `enter the details of the new user`

**Comment by MylesJarvis** [comment]
I'll merge this once we're ready to ship the DSI work - I need to ensure that the date is correct before this goes to prod.

File: data/service_updates/20240402.yml [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+  <p>When you add a new user to your school profile they can:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>advertise experience at your school</li>
+    <li>manage bookings and cancellations</li>
+    <li>add other people from your school</li>
+  </ul>
+  <h3>Adding a new user</h3>
+  <p>To add a new user:</p>
+  <ol class="govuk-list govuk-list--number">
+    <li>go to the school dashboard in your account</li>
```
Comment (MaxineCl) [inline data/service_updates/20240402.yml:19]: @MylesJarvis tiny thing. In a numbered bullet list each point should start with a capital letter and end in a full stop. https://design-system.service.gov.uk/styles/lists/#numbered-lists Sorry I should have flagged it earlier.

Comment (MylesJarvis) [reply data/service_updates/20240402.yml:19]: I'll sort this now, thank you!

**Pull request merged**